snap/snapcraft.yaml

```diff
@@ -213,130 +202,24 @@ parts:
       - lib/libdqlite*so*
       - lib/libraft*so*
       - lib/*/libuv*so*
-  rtrlib:
-    build-packages:
-      - cmake
-      - make
-      - gcc
-      - libssh-dev
-    stage-packages:
-      - libssh-4
-      - zlib1g
-    prime:
-      - usr/local/lib/librtr.so*
-      - usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR/libssh.so*
-    source: https://github.com/rtrlib/rtrlib.git
-    source-type: git
-    source-tag: v0.8.0
-    plugin: cmake
-    cmake-parameters:
-      - -DCMAKE_BUILD_TYPE=Release
-  libyang:
-    build-packages:
-      - cmake
-      - make
-      - gcc
-      - libpcre2-dev
-    stage-packages:
-      - libpcre2-8-0
-    source: https://github.com/CESNET/libyang.git
+
+  bird:
+    plugin: autotools
+    source: https://gitlab.nic.cz/labs/bird.git
     source-type: git
-    source-tag: v2.1.128
-    plugin: cmake
-    cmake-parameters:
-      - -DCMAKE_INSTALL_PREFIX:PATH=/usr
-      - -DENABLE_LYD_PRIV=ON
-      - -DENABLE_CACHE=ON
-      - -DCMAKE_BUILD_TYPE:String="Release"
-  frr:
-    after: [rtrlib,libyang]
+    source-tag: v3.1.4
     build-packages:
       - gcc
       - autoconf
-      - automake
       - libtool
-      - make
-      - gawk
-      - libreadline-dev
-      - libelf-dev
-      - texinfo
-      - libncurses5-dev
-      - texlive-latex-base
-      - texlive-latex-recommended
-      - libcap-dev
-      - imagemagick
-      - ghostscript
-      - groff
-      - libpcre3-dev
-      - chrpath
-      - pkg-config
-      - libjson-c-dev
-      - libc-ares-dev
       - bison
       - flex
-      - python3-dev
-      - libprotobuf-c-dev
-      - protobuf-c-compiler
-      - python3-sphinx
-    stage-packages:
-      - coreutils
-      - iproute2
-      - logrotate
-      - libjson-c5
-      - libc-ares2
-      - libatm1
-      - libprotobuf-c1
-      - libdb5.3
-      - libacl1
-      - libattr1
-      - libaudit1
-      - libselinux1
-      - libxtables12
-    plugin: autotools
-    source: https://github.com/FRRouting/frr.git
-    source-type: git
-    source-tag: frr-10.1.1
-    autotools-configure-parameters:
```

Comment:
Let's keep path configuration that would be compatible with switching to the .deb when we're ready for that. That would also get rid of these long `usr/local/` paths.

https://salsa.debian.org/debian/bird3/-/blob/debian/experimental/debian/rules?ref_type=heads#L12-L15